src/rtp.cpp

```diff
@@ -0,0 +1,163 @@
+/*
+ * This file is part of EasyRPG Player.
+ *
+ * EasyRPG Player is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * EasyRPG Player is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with EasyRPG Player. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <array>
+#include <cassert>
+#include <cstring>
+#include "rtp.h"
+
+namespace RTP {
+    extern const char* rtp_table_2k[][5];
+    extern const char* rtp_table_2k3[][7];
+
+	constexpr int num_2k_rtps = 4;
+	constexpr int num_2k3_rtps = 6;
```

Comment:
You could declare the second array bound with `num_2k_rtps + 1` etc..

Reply:
good idea, this way it generates a compiler error when these numbers are not updated :)